templates/contest/ranking-table.html

```diff
@@ -21,6 +21,16 @@
 {% endblock %}
 
 {% block user_data %}
+    {% if not contest.ended %}
+        {% if user.participation.ended %}
+            <div class="start-time active">
+                {{ relative_time(user.participation.start, abs=_('Started on {time}'), rel=_('Started {time}')) }}
+            </div>
+        {% else %}
+            <div class="start-time">{{ _('Participation ended.') }}</div>
```

Comment:
Are you sure you didn't flip the if and else conditions?

Reply:
Fixed :notlikeblob:.